src/ui/ResultActions/ResultActionsMenu.ts

```diff
@@ -0,0 +1,126 @@
+import { Component } from '../Base/Component';
+import { ComponentOptions } from '../Base/ComponentOptions';
+import { IResultsComponentBindings } from '../Base/ResultsComponentBindings';
+import { $$ } from '../../utils/Dom';
+import { IQueryResult } from '../../rest/QueryResult';
+import { Assert } from '../../misc/Assert';
+import { Initialization } from '../Base/Initialization';
+import * as _ from 'underscore';
+import { exportGlobally } from '../../GlobalExports';
+import 'styling/_ResultActionsMenu';
+
+export interface IResultActionsMenuOptions {
+  openOnMouseOver?: boolean;
+}
+
+/**
+ * The ResultActionsMenu is a small floating slack-like result actions menu.
+ * It is designed to contain "buttons" that execute an action on a result when pressed.
```

Comment:
Is there any limitation on the buttons that can be displayed here ?

I'm guessing it's relatively limited in the standard UI. Is there anything else than the quickview button that can be added here ?

If someone wants to create a custom component for this, what kind of guideline should they follow ?

Reply:
Remove this and use what I commented before.